scripts/gulp/utils/childProcessUtils.ts

```diff
@@ -269,3 +265,14 @@ function streamHandler (cp: ChildProcess, config: StreamHandlerConfig) {
 
   return dfd.promise
 }
+
+const isWin = process.platform === 'win32'
+
+/**
+ * Pretreat commands to make them compatible with windows
+ * @param command
+ * @returns
+ */
+export function winSpawn (command: string): string {
+  return `${(command).replace(/\//g, '\\')}${isWin ? '.cmd' : ''}`
+}
```

Comment:
This seems like a strange name since it works on *nix and Windows

Reply:
I agree `universalSpawn` woudl be better